content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -205,8 +205,10 @@ by the Kubernetes cluster, and then present that certificate to the Kubernetes A
 ### Create private key
```

Comment:
Consider whether this title is still appropriate if we're adding more detail about the (X.509) CSR.

The next section, “Create CertificateSigningRequest”, is also confusing, because you take the certificate signing request and you use it to make a (Kubernetes) CertificateSigningRequest.

Reply:
@sftim So, what things you would like me to change?